### ISSUE TITLE
Replace PDFView with webview in workshop seminar page

### ISSUE THREAD

**Issue by HeySitam** [ByteMonk-GCECT/GCECT-APP#44]
### GSSoC-22

yes

### Type of Issue

New Feature

### Description

On the workshop seminar page, there has a pdf view to show the pdf. This pdfview is needed to be replaced with a webview to load the same pdf.

![replace_pdfview_with_webview_workshop_seminar_page](https://user-images.githubusercontent.com/76570320/158215876-714af194-40bb-414b-94bd-60d7756e33c4.jpg)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by priyanshuborole** [comment]
can i work on this issue?

**Comment by HeySitam** [comment]
sure

**Comment by HeySitam** [comment]
any update?

**Comment by priyanshuborole** [comment]
yes i am working on it , finding another method instead of loading it with google docs because of limit error

**Comment by HeySitam** [comment]
Great🤩🤩.

**Comment by priyanshuborole** [comment]
Can you provide the workshop seminar pdf link with a .pdf extension? @sitamadex11

**Comment by HeySitam** [comment]
I think the link is already there in the code.

**Comment by priyanshu942** [comment]
Please assign me this issue under GSSOC'23.
